lib/plan_executor/config.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require 'hocon'
+require 'bolt/error'
+
+module PlanExecutor
+  class Config
+    CONFIG_KEYS = ['host', 'port', 'modulepath', 'ssl-cert', 'ssl-key',
+                   'ssl-ca-cert', 'ssl-cipher-suites', 'loglevel', 'logfile',
+                   'whitelist', 'workers', 'cache-dir',
+                   'file-server-conn-timeout', 'file-server-uri'].freeze
+
+    ENV_KEYS = ['ssl-cert', 'ssl-key', 'ssl-ca-cert', 'loglevel',
+                'workers', 'file-server-conn-timeout', 'file-server-uri'].freeze
+
+    DEFAULTS = {
+      'host' => '127.0.0.1',
+      'port' => 67526,
```

Comment:
This isn't a legal port on many systems (it's > 65535). We should probably just make it adjacent to the bolt-server port for simplicity.